src/components/image-contentBlock/image-contentBlock.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Only use kebap-case filenames, don't mix with camel-case: `image-content-block.*`

Reply:
I am sorry, I have forgotten it another time...
Fixed it, and will remember it